### DISCUSSION TITLE
Write Release Date to RELEASETIME field

### DISCUSSION THREAD

**Discussion by thedjnamedves** [Marekkon5/onetagger#186] [Ideas]
Currently I notice that Onetagger writes the release date to the YEAR field rather than the RELEASETIME field. 

Can this write to both or just write the to RELEASETIME field.

**Comment by Marekkon5** [comment]
If you enable release date, then 1T will write to RELEASEDATE IF the date is available, otherwise it will write to YEAR. (also iirc this only applies to ID3v2.3, otherwise it's always the correct release date tag)

**Comment by thedjnamedves** [comment]
Release date is enabled but it writes to the YEAR field rather than the RELEASEDATE. Is it writing to TDRC?
Using MP3tag to check the tags. I also checked with Kid3 and it was written to the YEAR field.

Using ID3.v2.4

**Comment by thedjnamedves** [comment]
Added some screenshots
![MP3tag](https://user-images.githubusercontent.com/125754300/219954102-7f64242c-c300-4e78-8197-aaa37a6b098c.png)
![RB668](https://user-images.githubusercontent.com/125754300/219954104-c21f5170-9659-40db-bce8-570b7c957131.png)

**Comment by Marekkon5** [comment]
Sorry for late reply, but:

Release Date:
ID3v2.3: TYER + TDAT
ID3v2.4: TDRC

**Comment by thedjnamedves** [comment]
That explains things. RELEASE DATE = TDRL but TDRC is used more in this place. https://wiki.hydrogenaud.io/index.php?title=Foobar2000:ID3_Tag_Mapping

I'll do some further checking on my side. Might have to adjust my workflow or do things manually.

And please don't say sorry. It's the weekend. People need to enjoy their free time!

**Comment by bascurtiz** [comment]
The majority of the DJ apps show the Year column - which is made out of TYER+TDAT when MP3 id3v2.3 or TDRC when MP3 id3v2.4.
RELEASETIME as you suggested isn't read by any of them.
I bet you use MusicBee (which is able to show more exotic tagfields)?

The gsheets below are linked within the help section of One Tagger, when clicking on the DJ app icons.

Rekordbox: https://docs.google.com/spreadsheets/d/1ALyk1WRTR_-FLq7kntdX2e4jM9ZARvieJAHz9-pjRQ8/edit?usp=sharing
Serato: https://docs.google.com/spreadsheets/d/1QAnFmRMhcNrM9yARNrjyIP5wif5h-36eSGMcXarXNsk/edit?usp=sharing
Traktor: https://docs.google.com/spreadsheets/d/1Atb26winoLgb_w_oX12npmZzqx3IVkCbkzn9Ug6R6B8/edit?usp=sharing
VirtualDJ: https://docs.google.com/spreadsheets/d/12oVBNt5Zs8fC9qId3M9jkRT9DYf3hdaaet0W1IQIuB4/edit?usp=sharing

**Comment by thedjnamedves** [comment]
Only use MP3Tag and RELEASETIME is read by Rekordbox, which is what I'm using. See my screenshots for the example.

it's true the DJ programs don't have the column of Release Date in the library view, but RB does have this in the information tab. Taktor as well.
And it can be selected as a parameter for intelligent playlists in RB.

**Comment by bascurtiz** [comment]
The column in Traktor shows Release Date, but that's just the name. It displays TYER+TDAT and TDRC.
So that makes Rekordbox unique displaying both Release Date and Year.
Hence we're a little conservative in implementing, when it's basically only one DJ app that can display both.
